package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "@react-native-community/async-storage": "^1.6.3"
+  }
+}
```

Comment:
Remove this file from the root of your project.

Reply:
## Removed